### PR TITLE
Remove duplicate keywords for autocompletion

### PR DESCRIPTION
**Pull request by dail8859**
Some of the API XML files had a lot of duplicate entries

### CODE REVIEW

File: PowerEditor/installer/APIs/actionscript.xml [hunk 1]
```diff
@@ -25,320 +22,211 @@
...
 		<KeyWord name="broadcastMessage" />
 		<KeyWord name="bullet" />
 		<KeyWord name="Button" />
 		<KeyWord name="call" />
-		<KeyWord name="call" />
 		<KeyWord name="callee" />
 		<KeyWord name="caller" />
 		<KeyWord name="Camera" />
 		<KeyWord name="capabilities" />
 		<KeyWord name="Capabilities" />
```
Comment (SinghRajenM) [inline PowerEditor/installer/APIs/actionscript.xml]: What about this? Does camelcase matters?

File: PowerEditor/installer/APIs/actionscript.xml [hunk 2]
```diff
@@ -25,320 +22,211 @@
...
 		<KeyWord name="enabled" />
 		<KeyWord name="END" />
-		<KeyWord name="END" />
 		<KeyWord name="endFill" />
 		<KeyWord name="ENTER" />
-		<KeyWord name="ENTER" />
-		<KeyWord name="enterFrame" />
 		<KeyWord name="enterFrame" />
 		<KeyWord name="eq" />
 		<KeyWord name="escape" />
```
Comment (SinghRajenM) [inline PowerEditor/installer/APIs/actionscript.xml]: Line 166 and 169 look identical.

File: PowerEditor/installer/APIs/actionscript.xml (hunk 1 above)
Comment (dail8859) [reply PowerEditor/installer/APIs/actionscript.xml]: I'm honestly not sure. I ran a simple regex to find duplicate lines. I am planning on looking into this eventually if case matters.

File: PowerEditor/installer/APIs/actionscript.xml (hunk 2 above)
Comment (dail8859) [reply PowerEditor/installer/APIs/actionscript.xml]: Again just ran a regex to find duplicates next to each other so it doesn't work if the list isn't properly sorted.

Comment (SinghRajenM) [reply PowerEditor/installer/APIs/actionscript.xml]: I feel this also should be removed.

image

**Comment by dail8859** [comment]
Will do. 
 
  On Sun, Dec 10, 2017 at 12:43 PM, Rajendra Singh<notifications@github.com> wrote:   
@SinghRajenM commented on this pull request.

In PowerEditor/installer/APIs/actionscript.xml:
>  		<KeyWord name="endFill" />
 		<KeyWord name="ENTER" />
-		<KeyWord name="ENTER" />
-		<KeyWord name="enterFrame" />
 		<KeyWord name="enterFrame" />
 		<KeyWord name="eq" />
 		<KeyWord name="escape" />

I feel this also should be removed.

**Comment by dail8859** [comment]
I would sort all the keyword and then remove duplicates with a regex but wanted to avoid alot of modifications. But I agree a programmatic way would probably be better in the end.

**Comment by MAPJe71** [comment]
Or maybe an XML Schema file?

**Comment by dail8859** [comment]
PR updated and now includes changes from #3979 (thanks to @SinghRajenM)

**Pull request merged**